src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSActionPlanServiceTest.kt

```diff
@@ -45,6 +45,11 @@ internal class SNSActionPlanServiceTest {
       mapOf(
         "actionPlanId" to UUID.fromString("77df9f6c-3fcb-4ec6-8fcf-96551cd9b080"),
         "submittedBy" to actionPlanSubmittedEvent.actionPlan.submittedBy!!.userName,
+        "referralId" to UUID.fromString("68df9f6c-3fcb-4ec6-8fcf-96551cd9b080"),
+        "referralReference" to "HAS71263",
+        "contractTypeName" to "Accommodation",
+        "primeProviderName" to "Harmony Living",
+        "actionPlanProbationUserUrl" to "https://base/url/action-plans/68df9f6c-3fcb-4ec6-8fcf-96551cd9b080",
```

Comment:
`https://base/url/action-plans/68df9f6c-3fcb-4ec6-8fcf-96551cd9b080` is this URL correct?